site/admin/theatres/create.php

```diff
@@ -0,0 +1,107 @@
+<?php
+include_once('init.php');
+PrintHeader('Create Theatre', 'Create Theatre');
+
+
+if(isset($_REQUEST['TheatreFormSubmitted'])){
+	//die("the form was submitted");
+	if(!@$_REQUEST['TheatreName']){
+		$FormErrors['TheatreName'] = "Theatre Name required";
+	}
+	if(!@$_REQUEST['TheatreDescription']){
+		$FormErrors['TheatreDescription'] = "Theatre must have a description";
+	}
+	if(!@$_REQUEST['Email'] || strpos($_REQUEST['Email'], '@') == false){
```

Comment:
Maybe this is on your list, but I'd suggest finding a more robust email validation approach. A keyword to search for is "regular expression"